app/layouts/Dashboard/index.js

```diff
@@ -1,16 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Navbar from 'Components/Navbar';
 import Sidebar from 'Components/Sidebar';
 
 import './style.scss';
 
 const Dashboard = ({ children, data }) => {
   if (!data) return null;
+  const [showSideBar, setSideBar] = useState(true);
+
+  const toggleSideBar = () => {
+    setSideBar(!showSideBar);
+  };
 
   return (
     <div className="dashboard-layout">
-      <Navbar />
-      <Sidebar />
+      <Navbar handleSideBar={toggleSideBar} />
+      {showSideBar && <Sidebar />}
```

Comment:
Use classnames to hide the sidebar and add animations. Refer to this: https://gogo-react.coloredstrategies.com/app/dashboards/default

Reply:
so you mean to wrap up in a div and use classes to toggle right?

Reply:
or to pass props to sidebar and using them toggle the sidebar component itself!

Reply:
pass state to the sidebar and let sidebar add or remove appropriate classes to the div it is rendering.

Reply:
Sure. I'll just add the functionality.

Reply:
![image](https://user-images.githubusercontent.com/35633575/82156881-aabbab00-989b-11ea-811c-0851f6d47c60.png)

How can i receive props here withRouter?

Reply:
I tried to receive props but couldn't correctly i gues..
![image](https://user-images.githubusercontent.com/35633575/82157024-9d52f080-989c-11ea-947f-a85c9e6f29cd.png)

### pressed 6 times
![image](https://user-images.githubusercontent.com/35633575/82157032-a9d74900-989c-11ea-9ddb-f71f66823863.png)

Reply:
I'll let you spend 2-3 more hours figuring out what is wrong. That's the fun of it.

Reply:
Done!